nservicebus/messaging/non-durable-messaging_rabbitmq-confirms_core_[,5].md

```diff
@@ -0,0 +1,3 @@
+The default behavior of sending messages via RabbitMQ is to wait for the publisher confirmation before proceeding. When non-durable messaging is used, the RabbitMQ transport skips the publisher confirmation step.
+
+This effectively means the blocking call to [WaitForConfirmsOrDie](http://www.rabbitmq.com/javadoc/com/rabbitmq/client/Channel.html#waitForConfirmsOrDie%28long%29) is skipped.
```

Comment:
I suppose so. My emphasis was on making the sure the content was correct for the current version vs. rewriting everything for the previous version of the content.